src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,54 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ResponseError';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  addingMovies: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ addingMovies }) => {
+  const [searchedValue, setSearchedValue] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [errorMassage, setErrorMassage] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchedValue(event.target.value);
+    setErrorMassage('');
+  };
+
+  const preparingMovie = (movieFromServer: MovieData): Movie => {
+    return {
+      title: movieFromServer.Title,
+      description: movieFromServer.Plot,
+      imgUrl: movieFromServer.Poster !== 'N/A'
+        ? movieFromServer.Poster
+        : 'https://via.placeholder.com/360x270.png?text=no%20preview',
```

Comment:
nice idea, but better to create constant with this link and use instead of string